vizro-core/changelog.d/20250811_102650_90609403+huong-li-nguyen_enable_single_actions_output.md

```diff
@@ -0,0 +1,45 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Highlights ✨
+
+- A bullet item for the Highlights ✨ category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX. ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+<!--
+### Removed
+
+- A bullet item for the Removed category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX. ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+### Added
+
+- Enable single string outputs in `Action` model. You can now specify `outputs="component_id"` instead of requiring `outputs=["component_id"]` for single outputs. ([#1334](https://github.com/mckinsey/vizro/pull/1334))
```

Comment:
```suggestion
- Enable single string outputs in `Action` model. You can now specify `outputs="component_id.property"` instead of `outputs=["component_id.property"]` for single outputs. ([#1334](https://github.com/mckinsey/vizro/pull/1334))
```

The `component_id` without `property` documentation is still WIP so I wouldn't mention it yet.